mat/cdense.go

```diff
@@ -29,9 +33,53 @@ func (m *CDense) Dims() (r, c int) {
 func (m *CDense) Caps() (r, c int) { return m.capRows, m.capCols }
 
 // H performs an implicit conjugate transpose by returning the receiver inside a
-// Conjugate.
+// ConjTranspose.
 func (m *CDense) H() CMatrix {
-	return Conjugate{m}
+	return ConjTranspose{m}
+}
+
+// T performs an implicit transpose by returning the receiver inside a
+// CTranspose.
+func (m *CDense) T() CMatrix {
+	return CTranspose{m}
+}
+
+// ConjElem calculates the element-wise conjugate of a and stores the result
+// in the receiver.
+// ConjElem will panic if m and a do not have the same dimension unless m is empty.
+func (m *CDense) ConjElem(a CMatrix) {
```

Comment:
Even though I proposed it, I'm wondering if just `Conj` would be enough or better now that Conjugate is renamed to ConjTranspose. The naming and effect would then be consistent with `cmplx.Conj`.

Reply:
Thanks, this was what I was thinking too.